habushu-mixology-consumer/pom.xml

```diff
@@ -39,6 +39,7 @@
                             <operatorAndVersion>${project.version}</operatorAndVersion>
                         </packageDefinition>
                     </managedDependencies>
+                    <testLintDisabledChecker>C,R,W,E0102</testLintDisabledChecker>
```

Comment:
I think this comment seems targeted for a specific project - recommend noting the general beenfit of this vice a project-specific use case (Seems like this could still be useful outside whatever use can you are thinking of specifically).

Reply:
This looks like it's specifically referring to the `step_impl` glue code for the Behave BDD scenarios of Mixology.  But since Habushu pushes Behave as the preferred testing framework, and Behave encourages the use of a redefined `step_impl` function, we should definitely make this the _default_ configuration.